packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -508,3 +510,16 @@ export function kill(proc: ChildProcess) {
 		(resolve) => proc.pid && treeKill(proc.pid, "SIGINT", () => resolve()),
 	);
 }
+
+export function packWrangler(): string {
+	const pathToWrangler = path.resolve(__dirname, "../../wrangler");
+	execSync("pnpm pack --pack-destination ./pack", { cwd: pathToWrangler });
```

Comment:
Unfortunately, this doesn't quite do what we need.
This will pack Wrangler, yes, but it will change the local Wrangler dependencies to published ones. For example,

```
"miniflare": "workspace:*",
```

becomes

```
"miniflare": "3.20241230.2",
```

which means that when C3 installs this version of Wrangler it will pull in the published miniflare, instead of the locally built one.

This is notably problematic because one of the reasons we wanted to do this was that the transitive dependency on `@cloudflare/unenv-preset` that was added did not cause a failure in the C3 tests.

Reply:
The vitest-pool-workers package has the same problem.
This is how it gets around it: https://github.com/cloudflare/workers-sdk/blob/f3c040016bad4fd4c7a222e9d5f74a1a6207529e/packages/vitest-pool-workers/test/global-setup.ts

Reply:
I wonder if we could leverage turbo to do that more automatically?